src/main.rs

```diff
@@ -110,9 +117,22 @@ fn print_page(path: &Path) -> Result<(), String> {
     );
     let reader = BufReader::new(file);
 
+    // Look up config file, if none is found fall back to default config.
+    let config = match Config::new() {
+        Ok(config) => config,
+        Err(ConfigError(msg)) => {
+            println!("Failed to create config: {}", msg);
+            process::exit(1);
+        }
+        Err(_) => {
+            println!("Unknown error while creating config");
+            process::exit(1);
+        }
```

Comment:
You could use [`eprintln!`](https://stackoverflow.com/questions/27588416/how-to-send-output-to-stderr#27590832) in those two cases. Same thing for the other errors below that result in `process::exit(1)`.